celery/worker/request.py

```diff
@@ -535,21 +533,4 @@ def execute_using_pool(self, pool, **kwargs):
             self._apply_result = maybe(ref, result)
             return result
 
-        def on_success(self, failed__retval__runtime, **kwargs):
```

Comment:
doesn't removing this effect any public API?

Reply:
No I don't think so, my understanding is the super class will be the [Request](https://github.com/celery/celery/blob/master/celery/worker/request.py#L68) class that already has this method defined so its not being removed.

Reply:
I guess if the user provides a custom request that doesn't derive from [Request](https://github.com/celery/celery/blob/master/celery/worker/request.py#L68) then the API is changed, not sure the value of this use case.